### PR TITLE
feature: slider component

### PR DESCRIPTION
**Pull request by sjkayle**
New slider component in chlorophyll and angular

ezgif-1-83791a7369

### References
[Design Library](https://designlibrary.sebgroup.com/components/component-slider)
[Github Issue](https://github.com/sebgroup/green/issues/762)

### CODE REVIEW

**Comment by sebopensource** [comment]
👋 Thanks for creating a pull request! 🚀 Checkout the storybooks we've created for it: <ul><li><a target="_blank" href="https://sebgroup.github.io/green/pr-761/chlorophyll/">chlorophyll</a></li><li><a target="_blank" href="https://sebgroup.github.io/green/pr-761/angular/">angular</a></li></ul>

File: libs/angular/src/lib/slider/slider.component.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  onTouchedFn?: VoidFunction
+  style: SliderStyle = {}
+
+  ngOnInit(): void {
+    this.setTrackBackground()
+  }
+
+  onFocus(): void {
+    this.sliderTouch.emit(true)
+    this.onTouchedFn && this.onTouchedFn()
```
Comment (Maximilian-R) [inline libs/angular/src/lib/slider/slider.component.ts:54]: The [abstract control](https://angular.io/api/forms/AbstractControl) interface notes that `A control is marked touched once the user has triggered a blur event on it`

File: libs/chlorophyll/scss/components/slider/_index.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  &:disabled {
+    @each $browser in $browser-thumb-classes {
+      &::#{$browser} {
+        @include thumb-disabled-styles();
+      }
+    }
+  }
+}
+
+.ngg-slider {
```
Comment (Maximilian-R) [inline libs/chlorophyll/scss/components/slider/_index.scss:41]: The ngg namespace refers to "Angular Green" and should probably not be used in chlorophyll. I think it should be swapped with "gds", can you confirm @vsjolander ?

File: libs/angular/src/lib/slider/slider.component.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<div class="ngg-slider-label-container">
+  <label [attr.for]="name">Slider label text in one line</label>
```
Comment (Maximilian-R) [inline libs/angular/src/lib/slider/slider.component.html:2]: Label text should probably be an Input? :)

**Review by Maximilian-R — CHANGES_REQUESTED** [review]

File: libs/chlorophyll/scss/components/slider/_index.scss (hunk 2 above)
Comment (sjkayle) [reply libs/chlorophyll/scss/components/slider/_index.scss:41]: I think you're right. My bad. Will rename my css classes :D

File: libs/angular/src/lib/slider/slider.component.html (hunk 3 above)
Comment (sjkayle) [reply libs/angular/src/lib/slider/slider.component.html:2]: Totally missed this. Will change this one too!

**Comment by sjkayle** [comment]
Hi @Maximilian-R I've pushed some changes based on the comments you gave :D Kindly review again. Thank you!

**Review by Maximilian-R — APPROVED** [review]
Awesome, great work @sjkayle 😃🥳

**Comment by daneriseb** [comment]
Where is this documented? I can't find it here https://sebgroup.github.io/green/latest/chlorophyll/

**Comment by sjkayle** [comment]
@daneriseb Hi, it seems that a deployment was not yet triggered for the latest main branch, hence it isn't published yet to the that web page. :)

cc: @Maximilian-R

**Comment by daneriseb** [comment]
Now it's there! How does one use this with javascript? I don't see any documentation for that (it only says "Note: use javascript to bind values to textbox and to control track background.").

Are there any plans on making a React component for this? We are gonna need one soon, so I'm wondering if we need to build it ourselves.

**Comment by sjkayle** [comment]
For now we only have an angular adaptation of this component. You may refer [here](https://sebgroup.github.io/green/latest/angular/?path=/docs/components-slider--default).

In the angular component, you can also see how we use js to control the track background style of the slider:
```
const _value = ((this.value - this.min) / (this.max - this.min)) * 100
this.style.background = getSliderTrackBackground(_value)
```

**Comment by sjkayle** [comment]
@daneriseb a React version of this component is in the works! :D

**Pull request merged**